src/backend/TrafficCourts/Citizen.Service/Controllers/DisputesController.cs

```diff
@@ -220,4 +221,72 @@ public async Task<IActionResult> VerifyEmailAsync([Required] string token, Cance
             throw;
         }
     }
+
+    /// <summary>
+    /// Submits an update request for a Disputant's contact information.
+    /// </summary>
+    /// <param name="guidHash">A hash of the noticeOfDisputeGuid.</param>
+    /// <param name="disputantContactInformation">The requested fields to update.</param>
+    /// <param name="cancellationToken"></param>
+    /// <returns></returns>
+    /// <response code="200">The Dispute is updated.</response>
+    [HttpPut("/api/dispute/{guidHash}/contact")]
+    [ProducesResponseType(StatusCodes.Status200OK)]
+    [ProducesResponseType(StatusCodes.Status400BadRequest)]
+    [ProducesResponseType(StatusCodes.Status404NotFound)]
+    public async Task<IActionResult> UpdateDisputeContactInfoAsync([Required] string guidHash, [FromBody] DisputantContactInformation disputantContactInformation, CancellationToken cancellationToken)

```

Comment:
We are not being consistent with these parameter names. For example in `ResendEmailAsync` we use `string uuidHash`. Additionally, when we return the bad request, we are using a different name, `Invalid noticeOfDisputeGuid`. The bad request data is returned to the user and the front end may display it.

Reply:
Done. Agreed, we should rename the uuidHash to guidHash in the other endpoint since we are using GUIDs, not UUIDs.